### PR TITLE
Added instructions to get hugging face token

### PR DESCRIPTION
**Pull request by RodrigoLPA**

### CODE REVIEW

File: docs/validator.md [hunk 1]
```diff
@@ -84,39 +87,74 @@ pip install torch
...
 docker compose up --detach
 ```
----
 
-# Running the Validator
+## Obtaining your Hugging Face token
+
+The dataset for code, `The Stack V1-dedup`, requires a **Hugging Face access token**. Follow these steps to obtain and configure one:
+
+### Step 1: Get Your Hugging Face Access Token
```
Comment (cryptal-mc) [inline docs/validator.md:104]: I think that they first need to go to the dataset page and enter their email address and submit to unblock access

**Review by cryptal-mc — CHANGES_REQUESTED** [review]
One small change requested

File: docs/validator.md (hunk 1 above)
Comment (RodrigoLPA) [reply docs/validator.md:104]: added it now!

**Review by cryptal-mc — APPROVED** [review]

**Comment by cryptal-mc** [comment]
replaced by #195

**Pull request closed without merging**